docs/Docs_Manual_QA.md

```diff
@@ -15,6 +15,8 @@ XXX Test Failed: <link to the test results in your GitHub repository>, <link to
 </br>
 
 
+AI_WORKFLOW_SECTION (DEV_ENV) Test Passed: https://github.com/Daddypastor/mindsdb_test
```

Comment:
Please changeAI_WORKFLOW_SECTION (DEV_ENV) to  Create AI Engines section using SQL